src/store/metadata/stub/content/editions/additions-to-the-housing-stock_2022-2023.json

```diff
@@ -0,0 +1,163 @@
+  
+        ],
+        "spatial_coverage": "E92000001",
+        "temporal_resolution": [
+          "P1Y"
+        ],
+        "temporal_coverage": {
+          "start": "2022-01-01T00:00:00Z",
+          "end": "2023-12-31T00:00:00Z"
+        },
+      "versions_url": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions/2022-2023/versions",
+      "versions": [
+        {
+          "@id": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions/2022-2023/versions/1",
+          "issued": "2023-05-05T08:59:11Z"
+        }
+      ],
+      "next_release": "2024-04-19T09:30:00Z",
+      "table_schema": {
+        "columns": [
+          {
+              "name": "Period",
+              "datatype": "string",
+              "titles": "Period",
+              "description": "The time period associated with the observation."
+            },
+            {
+              "name": "AREACD",
+              "datatype": "string",
+              "titles": "Statistical Geography",
+              "description": "The geography code associated with the observation."
+            },
+            {
+              "name": "AREANM",
+              "datatype": "string",
+              "titles": "Area Name",
+              "description": "The name assiciated with The geography code "
+            },
+            {
+              "name": "Geography",
+              "datatype": "string",
+              "titles": "Geography Type",
+              "description": "The categorisation of the geography code"
+            },
+            {
+              "name": "Variable Name",
+              "datatype": "string",
+              "titles": "Variable Name",
+              "description": "Which ESS variable is being measured"
+            },
+            {
+              "name": "Indicator", 		
+              "datatype": "string",
+              "titles": "Indicator",
+              "description": "Which ESS indicator is being measured"
+            },
+            {
+              "name": "Mission",
+              "datatype": "string",
+              "titles": "Mission",
+              "description": "Which ESS mission number this dataset is fulfilling"
+            },
+            {
+              "name": "Category",
+              "datatype": "string",
+              "titles": "Category",
+              "description": "Which ESS category the dataset belongs to"
+            },
+            {
+                "name": "Observation",
+                "datatype": "integer",
```

Comment:
datatype should be `"decimal"`, like in `apprenticeships-achievements_2023-05.json`